docs/questions/query-builder/expressions/now.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Now
+---
+
+# Now
+
+`now` returns the current datetime using your Metabase [report timezone](../../../configuring-metabase/localization.md#report-timezone).
+
+## Creating conditional logic using the current date or time
+
+Let's say you have some project data, and you want to add a status column for each task. We'll assume today's date and time is November 22, 2022, 12:00:00.
+
+| Task     | Start                       | Deadline                    | Status          |
+|----------|-----------------------------|-----------------------------|-----------------|
+| Draft    | November 1, 2022, 12:00:00  | November 30, 2022, 12:00:00 | In progress     |
+| Review   | November 15, 2022, 12:00:00 | November 19, 2022, 12:00:00 | Needs extension |
+| Edit     | November 22, 2022, 12:00:00 | November 22, 2022, 12:00:00 | DUE RIGHT NOW!  |
+
+To mark a task in progress, you'd use the expression:
+
+```
+now >= [Start] AND now < [Deadline] 
+```
+
+To check if you need to ask for an extension:
+
+```
+now >= [Start] AND now >= [Deadline]
+```
+
+If you're looking for an adrenaline rush (and you have real-time data), you can flag the tasks that are due _right this second_:
+
+```
+now = [Deadline]
+```
+
+To set up the **Status** column that combines all three situations above, you'd wrap everything in a `case` expression:
+
+```
+case(now >= [Start] AND now < [Deadline], "In progress",
+     now >= [Start] AND now >= [Deadline], "Needs extension",
+     now = [Deadline], "DUE RIGHT NOW!")
+```
+
+## Accepted data types
+
+| [Data type](https://www.metabase.com/learn/databases/data-types-overview#examples-of-data-types) | Works with `now`  |
+| ----------------------- | -------------------- |
+| String                  | ❌                   |
+| Number                  | ❌                   |
+| Timestamp               | ✅                   |
+| Boolean                 | ❌                   |
+| JSON                    | ❌                   |
+
+This table uses `timestamp` and `datetime` interchangeably. If your dates and times are stored as strings or numbers in your database, you can [cast them to datetimes](../../../data-modeling/metadata-editing.md#casting-to-a-specific-data-type) from the Data Model page.
```

Comment:
Do we need to get into datetimes with timezones vs without?

Reply:
Yes, I think it matters. But I think it's sufficient to state that for DBMSs that support timestamps with time zones, `now` is a timestamp with a time zone that is the reporting time zone. For others, it's a timestamp (without time zone) in the  reporting time zone.

Reply:
It's mostly implied in the decscription, but it's not crystal clear
https://github.com/metabase/metabase/pull/29182/files#diff-2d5b6a8a31092835d5e0c3a68db37209bd715411ab0644950fb7268bb3b88fe8R436